### ISSUE TITLE
DTMF tone not working

### ISSUE THREAD

**Issue by githubmss** [rajeshincorp/SendDtmfToneOverActiveCall#1]
i just imported your code in my studio , but i am unable to resolve the IInCallAdapter class as I am not able to download the java class from your given link 

http://friper.in/APIs/InCallAdapter.java

**Able to get Jar from below mentioned link** :- 

http://grepcode.com/snapshot/repository.grepcode.com/java/ext/com.google.android/android/5.1.1_r1/

**Also as you mentioned in the Work flow :-** 

When you will launch the app you will be landed to a page where you can enter a phone number and can press call button to make a phone call. Pressing call button will launch default InCall app[or Telephone app] that will make the phone call for you. Immediately the InCall app will go into the background and our custom dialer will appear on the screen. Now you will be able to use this custom dialer to send DTMF tone over the active phone call in the background. That's it.

So, we can divide the project into two parts
1) Sending the call into the background ---- Achieved.
2) Sending the DTMF tone over an active call ---- **Not Achieved Yet**. :- Is this Achieved in this demo now ?

**Able to see your DTMF tone keyboard but i am not able hear DTMF sound , kindly Let me know if it works or it i snot possible** ?

Here is the link of your working api (DTMF tone not working ) build generated via my pc

link :- https://www.dropbox.com/s/r12qewmz659bf2j/DTMF_demo.apk?dl=0

Please help me out , as i have exactly same requirement as you listed in your workflow.

Thanks

**Comment by rajeshincorp** [comment]
No,  '2. Sending the DTMF tone over an active call' is still not achieved and hence not in the demo app. I don't know whether it is possible or not. Sorry to say but I have abandoned this project about 20 months ago and 'friper.in' is expired now. Here is the files..
[Files.zip](https://github.com/rajeshincorp/UD_SendDtmfToneOverActiveCall/files/1424281/Files.zip)

It may help. Good luck :)

**Comment by githubmss** [comment]
Thanks for your reply man , can't we can do something live to play sound using player while person is on active call and type any key from DTMF keyboard ?

**Comment by rajeshincorp** [comment]
Yes, we can do that too, but that will not be so efficient, means DTMF tones will not be that clear on the other side of the call because of the positions of speaker and microphone in our phone, they are usually apart from each other hence less sound will reach to the microphone with added noise. However, it would be a weird way of doing this, but I guess will work to some extent. Best of luck :)
